README.md

```diff
@@ -142,3 +142,9 @@ Running with dbt=0.14.2
 06:35:33 + 2 of 478 Analyzing "analytics"."customer_payments"
 06:35:33 + 2 of 478 Finished "analytics"."customer_payments" in 0.28s
 ```
+
+The command can also be run with optional parameters to exclude schemas, either with exact or regex matching. This can be useful for cases where the models (or the schemas themselves) tend to be short-lived and don't require vacuuming. For example:
+```
+$ dbt run-operation redshift_maintenance --args '{exclude_schemas: ["looker_scratch"], exclude_schemas_like: ["sinter_pr_%"]}'
```

Comment:
My only comment here is that `_` is actually a reserved character -- it means "exactly one character of any type". Check this out:
```
$ select 'sinteripretty' ilike 'sinter_pr_%' ;
>
?column?
true

```

Weird huh?! Anyway the correct syntax here would be `'{exclude_schemas: ["looker\\_scratch"], exclude_schemas_like: ["sinter\\_pr\\_%"]}'`, but I am also happy to merge this as is!

Reply:
oy 🤦‍♂️ thanks for the catch! made the change (just for the regex matching though- i don't think the exact match needs it)